internal/communitymodules/modules_test.go

```diff
@@ -86,9 +94,125 @@ func Test_modulesCatalog(t *testing.T) {
 	})
 }
 
+func Test_ManagedModules(t *testing.T) {
+	t.Run("ok", func(t *testing.T) {
+		expectedResult := moduleMap{
+			"module1": row{
+				Name:    "module1",
+				Managed: "Managed",
```

Comment:
Please change it to `True` as the new value of the `MANAGED` field

Reply:
done